mongo_connector/namespace_config.py

```diff
@@ -227,10 +250,10 @@ def unmap_namespace(self, plain_target_ns):
             for src_name in src_name_set:
                 return src_name
         # The target namespace could also exist in the wildcard namespaces
-        for _, mapped in self._regex_map:
+        for _, namespace in self._regex_map:
             original_name = match_replace_regex(
-                namespace_to_regex(mapped.dest_name), plain_target_ns,
-                mapped.source_name)
+                namespace_to_regex(namespace.dest_name), plain_target_ns,
+                namespace.source_name)
```

Comment:
In future (especially larger) reviews, it would be easier to leave changes like this out, to keep the review as small as possible.